documentation/prometheus-mixin/jsonnetfile.json

```diff
@@ -0,0 +1,14 @@
+{
+    "dependencies": [
+        {
+            "name": "grafana-builder",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/kausalco/public",
```

Comment:
Would this be `https://github.com/monitoring-mixins/docs`? //CC @metalmatze

Reply:
I don't think so. Why?

Reply:
maybe @mxinden that is `https://github.com/grafana/jsonnet-libs` ?

Reply:
Yes, that seems to be the updated path. Thanks @lukmdo.